pkg/framework/utils.go

```diff
@@ -113,3 +113,7 @@ func GetControlPlaneHostAndPort(ctx context.Context, cl client.Client) (string,
 
 	return apiURL.Hostname(), int32(port), nil
 }
+
+func BoolPtr(v bool) *bool {
```

Comment:
Perhaps use the `ptr.To` pattern from [k8s.io/utils/ptr](https://pkg.go.dev/k8s.io/utils/ptr) instead?

Reply:
Updated